review_app_status.py

```diff
@@ -0,0 +1,137 @@
+# -*- encoding: utf-8
+
+import json
+import logging
+import os
+import time
+from enum import Enum
+
+import requests
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger("review_app_status")
+
+
+class BuildStates(Enum):
```

Comment:
Having an Enumeration with one value is redundant. Either add all other states or use a constant, such as `SUCCESS = "success"`.